src/CalcManager/Ratpack/num.cpp

```diff
@@ -433,20 +433,19 @@ void _divnum(PNUMBER* pa, PNUMBER b, uint32_t radix, int32_t precision)
             }
         }
 
-        if (digit)
+        if (digit && multiple)
```

Comment:
Is this `multiple` condition needed? Because this `if` does not check `multiple` before, and it was working correctly.
If there is any path that can cause `multiple` be `nullptr` in runtime, then we need to know why and handle it properly.